0.5.2rc0

calum-chamberlain opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

calum-chamberlain committed:
Merge branch 'main' of https://github.com/eqcorrscan/eqcorrscan-feedstock

calum-chamberlain committed:
0.5.2rc0

calum-chamberlain commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/17902030481](https://github.com/conda-forge/conda-forge-webservices/actions/runs/17902030481). Examine the logs at this URL for more detail.</sub>

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like I wasn't able to push to the `main` branch of `eqcorrscan`/`eqcorrscan-feedstock`. Did you check the "Allow edits from maintainers" box?

**NOTE**: Our webservices cannot push to PRs from organization accounts or PRs from forks made from organization forks because of GitHub permissions. Please fork the feedstock directly from conda-forge into your personal GitHub account.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/17901708484](https://github.com/conda-forge/conda-forge-webservices/actions/runs/17901708484). Examine the logs at this URL for more detail.</sub>

calum-chamberlain committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.20.07.44.53

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

calum-chamberlain commented:
Re-rendered locally - because this PR is coming from an org I am unable to allow edits from maintainers.

Note we expect the linter to fail for this test release.
